pkg/engine/fetchit.go

```diff
@@ -226,50 +228,50 @@ func getMethodTargetScheds(targetConfigs []*TargetConfig, fetchit *Fetchit) *Fet
 			fetchit.allMethodTypes = append(fetchit.allMethodTypes, cleanMethod)
```

Comment:
I think this needs to be updated, and the Restart fn can range thru the Method.Kinds instead 
https://github.com/containers/fetchit/blob/5305b2d0061f79d67e55a1123a4a82bf78ba21e3/pkg/engine/fetchit.go#L82-#L84